crates/bevy_text/src/pipeline.rs

```diff
@@ -101,123 +101,110 @@ impl TextPipeline {
 
         Ok(TextLayoutInfo { glyphs, size })
     }
+}
 
-    pub fn create_text_measure(
-        &mut self,
+#[derive(Debug, Clone)]
+pub struct TextMeasureSection {
+    pub text: Box<str>,
+    pub scale: f32,
+    pub font_id: FontId,
+}
+
+#[derive(Debug, Clone, Default)]
+pub struct TextMeasureInfo {
+    pub fonts: Box<[ab_glyph::FontArc]>,
+    pub sections: Box<[TextMeasureSection]>,
+    pub text_alignment: TextAlignment,
+    pub linebreak_behavior: glyph_brush_layout::BuiltInLineBreaker,
+    pub min: Vec2,
+    pub max: Vec2,
```

Comment:
Ah just noticed this from the changelog, it's not a mistake that these were called `min_content_width` and `max_content_width`. The word `width` here isn't referring to the result but the constraint the size value is based on. `min_content_width` is the space (horizontal and vertical) that the text will occupy given a zero width constraint (in which case the text wraps at every breakpoint) and `max_content_width` is the space the text will occupy given an infinite width constraint (in which case the text next never wraps).